libavfilter/dnn/dnn_backend_common.h

```diff
@@ -148,4 +148,20 @@ DNNReturnType ff_dnn_start_inference_async(void *ctx, DNNAsyncExecModule *async_
  */
 DNNAsyncStatusType dnn_get_async_result(Queue *task_queue, AVFrame **in, AVFrame **out);
 
+/**
+ * Allocate input and output frames and fill the Task
+ * with execution parameters.
+ *
+ * @param task pointer to the allocated task
+ * @param exec_params pointer to execution parameters
+ * @param backend_model void pointer to the backend model
+ * @param input_height height of input frame
+ * @param input_width width of input frame
+ * @param ctx pointer to the backend context
+ *
+ * @retval DNN_SUCCESS if successful
+ * @retval DNN_ERROR if allocation fails
+ */
+DNNReturnType dnn_get_output(TaskItem *task, DNNExecBaseParams *exec_params, void *backend_model, int input_height, int input_width, void *ctx);
```

Comment:
function name: ff_*

and, dnn_get_output is not correct, since we just fill the task here and we do not get output yet, need a new function name.

Reply:
Yes, it allocates memory for the frames and fills the task. How about `ff_dnn_allocate_frame_and_fill_tasks`?

Reply:
let's make it more straightforward, how about ff_dnn_file_gettingoutput_task

Reply:
Yes, looks good. Thanks